ssl/quic/quic_port.c

```diff
@@ -803,10 +809,11 @@ static void cleanup_validation_token(QUIC_VALIDATION_TOKEN *token)
  *
  * @return 1 if validation token is filled successfully, 0 otherwise.
  */
-static int generate_retry_token(BIO_ADDR *peer, QUIC_CONN_ID odcid,
-                                QUIC_CONN_ID rscid, QUIC_VALIDATION_TOKEN *token)
+static int generate_token(BIO_ADDR *peer, QUIC_CONN_ID odcid,
+                          QUIC_CONN_ID rscid, QUIC_VALIDATION_TOKEN *token,
```

Comment:
Here there is another "token" type. So now we have `QTOK`, `QUIC_TOKEN` and `QUIC_VALIDATION_TOKEN`. Why is this?